es6-operators/main.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-unused-vars */
+const v1 = { value: 'Something' };
+const v2 = '';
+const v3 = undefined;
+const v4 = [3, 5, 7, 9];
+
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
+ * In one expression, log "v1 is truthy" to the console only if `v1` is truthy.
+ * In one expression, log "v2 is truthy" to the console only if `v2` is truthy.
+ * Expected: v1 message logs, v2 message does not.
+ */
+
+if (v1) {
+  console.log('v1 is truthy');
+}
+
+if (v2) {
+  console.log('v2 is truthy');
+}
```

Comment:
No `if` statements. This should be done in "one expression" and utilizing the `&&` operator